### ISSUE TITLE
Could not create SSL/TLS secure channel

### ISSUE THREAD

**Issue by Sam-Lin-MillersLab** [mikaelkrief/GitHub-Tools-vsts-extensions#7]
I follow your blog http://www.mikaelkrief.com/create-github-release-with-release-notes-in-vsts-pipeline/

but I got this error. Any idea? thanks!

C:\agent\_work\_tasks\GitHubRelease_1c3396d6-5af6-4ccc-8169-199ec6541d23\1.0.0\GitHubRelease.ps1 : The request was aborted: Could not create SSL/TLS secure channel.

**Comment by Sam-Lin-MillersLab** [comment]
I think it is because Github use TLS 1.2 now.

**Comment by marcwittke** [comment]
As workaround you can force the agent to use TLS 1.2 by adding some registry keys on your build agent:

```
# set strong cryptography on 64 bit .Net Framework (version 4 and above)
Set-ItemProperty -Path 'HKLM:\SOFTWARE\Wow6432Node\Microsoft\.NetFramework\v4.0.30319' -Name 'SchUseStrongCrypto' -Value '1' -Type DWord


# set strong cryptography on 32 bit .Net Framework (version 4 and above)
Set-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\.NetFramework\v4.0.30319' -Name 'SchUseStrongCrypto' -Value '1' -Type DWord 
```

don't forget to restart the agent after applying those patches.

**Comment by mikaelkrief** [comment]
@Sam-Lin-MillersLab , @marcwittke I published a new version of the extension that contain a fix

**Comment by mikaelkrief** [comment]
fixed in #8 and #9
